sensum-pwa/src/app/App.res

```diff
@@ -1,44 +1,132 @@
-
 @react.component
 let make = () => {
-  
-  Sensations.getSensationsLength()->Promise.then(index => {
+  Sensations.getSensationsLength()
+  ->Promise.then(index => {
     Js.Console.log(index)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.log(err)
     Promise.resolve()
-  })->ignore
-  
+  })
+  ->ignore
+
   // Try with this instead, and check the COMPILE time error:
   // Sensations.getSensationByIndex(~index=1000)->Promise.then(sensation => {
-  Sensations.getSensationByIntIndex(~index=1000)->Promise.then(sensation => {
+  Sensations.getSensationByIntIndex(~index=1000)
+  ->Promise.then(sensation => {
     Js.Console.log(sensation)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.error(err)
     Promise.resolve()
-  })->ignore
-  
+  })
+  ->ignore
+
   let s: Sensations.sensation = {
     author: "mk",
-    message: "los Granujas sean unidos porque esa es la ley 1ra"
+    message: "los Granujas sean unidos porque esa es la ley 1ra",
   }
-  
-  Sensations.newSensation(~s=s)->Promise.then(confirmedTx => {
+
+  Sensations.newSensation(~s)
+  ->Promise.then(confirmedTx => {
     Js.Console.log(confirmedTx)
     Sensations.getLatestSensation()
   })
   ->Promise.then(sensation => {
     Js.Console.log(sensation)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.log(err)
     Promise.resolve()
-  })->ignore
-  
-  <div>
-    <h1 className="text-4xl font-bold"> {"Sensum PWA"->React.string} </h1>
-    <p className="text-2xl"> {"viteJs + react + rescript + tailwindcss"->React.string} </p>
+  })
+  ->ignore
+
+  <div className="bg-black w-full h-full">
+    <Navbar leftComponent={<GearIcon />}/>
+    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
```

Comment:
How do you write this in Rescript like JS iteration?

```
{[1,2,3,4,5,6].map(el => 
 <div key={el}>
  <h1 className="text-4xl text-purple-50"> {el} </h1>
 </div>
)}
```

Reply:
good one. Probably with something like: https://rescript-lang.org/docs/manual/latest/api/belt/list#map

Reply:
I will add it!